skyverge/wc-jilt-promotions
# Implement render_recover_carts_modal

# Summary

This PR implements the `Orders::render_recover_carts_modal` method.

### Story: [CH 61111](https://app.clubhouse.io/skyverge/story/61111/implement-render-recover-carts-modal)
### Release: #3 

## QA


- Remove Jilt for WooCommerce
- Pick a plugin with Jilt Promotions like WooCommerce Memberships
- Update the `skyverge/wc-jilt-promotions` version to `dev-ch61111/implement-render-recover-carts-modal` in the `composer.json` file
- Run `composer update skyverge/wc-jilt-promotions`
- Run `delete from wp_usermeta where meta_key = '_sv_wc_jilt_enabled_messages';`
- Make sure to have at least 10 orders or tweak `Orders::get_orders_count()` to return 11 and `get_orders_revenue()` to return 110 in `.../woocommerce-memberships/vendor/skyverge/wc-jilt-promotions/src/Admin/Orders.php`
- Open WooCommerce > Orders
![image](https://user-images.githubusercontent.com/4955242/90275445-b423c780-de38-11ea-9798-6a6afb3607a9.png)
- [x] The Abandoned Carts view is added
- Click on the Abandoned Carts view
![image](https://user-images.githubusercontent.com/4955242/90290963-37531680-de55-11ea-9c76-3869a29d04cd.png)
- [x] The popup shows 4 carts and $40.00 recovered revenue per month (if you're returning 11 orders and 110 order revenue), [formula](https://docs.google.com/document/d/1hm_uRWeTqrN2SiqpwGJleEBLbknZ0GsEgmVxhB8eKJQ/edit#bookmark=id.obb3njygxqqw)
- [x] The learn more link opens [this URL](https://www.skyverge.com/go/abandoned-cart-recovery) in a new tab
Click on the X to close the modal
- [x] The Abandon Carts view link is gone
- Run `delete from wp_usermeta where meta_key = '_sv_wc_jilt_enabled_messages';`
- Open WooCommerce > Orders
- Click on the Abandoned Carts view
- Click on I want to recover carts
- [x] Jilt is installed
- [x] The Abandon Carts view is not visible in WooCommerce > Orders anymore

## Before merge
